### PR TITLE
Add Eon95

### PR DESCRIPTION
**Pull request by Maartenwut**
## Description

Adds the Eon95 PCB with VIA keymap.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/maartenwut/eon95/eon95.c [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	// runs once when the firmware starts up
+	matrix_init_user();
+	led_init_ports();
+};
+
+void matrix_scan_kb(void) {
+	// put your looping keyboard code here
+	// runs every cycle (a lot)
+	matrix_scan_user();
+};
```
Comment (drashna) [inline keyboards/maartenwut/eon95/eon95.c:29]:
```suggestion
void matrix_init_kb(void) {
    // put your keyboard start-up code here
    // runs once when the firmware starts up
    matrix_init_user();
    led_init_ports();
}
```

**Comment by Maartenwut** [comment]
Merge, please? :)

File: keyboards/maartenwut/eon95/info.json [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+                {"x":0, "y":5.25, "w":1.25},
+                {"x":1.25, "y":5.25, "w":1.25},
+                {"x":2.5, "y":5.25, "w":1.25},
+                {"x":3.75, "y":5.25, "w":6.25},
+                {"x":10, "y":5.25},
+                {"x":11, "y":5.25},
+                {"x":12, "y":5.25},
+                {"x":13.25, "y":5.5},
+                {"x":14.25, "y":5.5},
+                {"x":15.25, "y":5.5}
```
Comment (fauxpark) [inline keyboards/maartenwut/eon95/info.json:332]:
```suggestion
                {"x":15.25, "y":5.5},
```

File: keyboards/maartenwut/eon95/info.json [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
```
Comment (fauxpark) [inline keyboards/maartenwut/eon95/info.json:223]:
```suggestion
                {"x":15.25, "y":5.5},
```

File: keyboards/maartenwut/eon95/info.json [hunk 4]
```diff
@@ -0,0 +1,338 @@
...
```
Comment (fauxpark) [inline keyboards/maartenwut/eon95/info.json:115]:
```suggestion
                {"x":15.25, "y":5.5},
```

File: keyboards/maartenwut/eon95/info.json [hunk 5]
```diff
@@ -0,0 +1,338 @@
...
+                {"x":16.5, "y":5.25},
+                {"x":17.5, "y":5.25},
+            ]
+        },
+        "LAYOUT_ansi": {
```
Comment (fauxpark) [inline keyboards/maartenwut/eon95/info.json:120]: This and the ISO layout are missing a key vs the LAYOUT macros. Split backspace?

**Comment by Maartenwut** [comment]
> 
> 
> Configurator preview doesn't like the info.json.

I did not know about the preview, that's very helpful!

**Pull request merged**